src/Components/prime-react-homepage/PrimeReactHomePage.tsx

```diff
@@ -156,17 +153,16 @@ export function PrimeReactHomePage() {
         <Sidebar visible={addVisibility} onHide={() => setAddVisibility(false)}>
           <Add toast={toast} setAddVisibility={setAddVisibility} />
         </Sidebar>
+        {console.log(employee)}
         <Sidebar
           visible={editVisibility}
           onHide={() => {
             setEditVisibility(false);
           }}
         >
-          <Edit
-            toast={toast}
-            setEditVisibility={setEditVisibility}
-            employee={employee}
-          />
+          <EmployeeContext.Provider value={employee}>
+            <Edit toast={toast} setEditVisibility={setEditVisibility} />
```

Comment:
setEditVisibility is not used anymore .. and this is the right approach. Visibility should be handled outside of the component

Reply:
It's used to set sidebar's "visible" prop to "false" resulting in hiding sidebar when submitting the form with new values.

Reply:
but is not used in the Edit component, remove it from there